containers/java-postgres/.devcontainer/docker-compose.yml

```
@@ -0,0 +1,52 @@
+version: '3.8'
+
+volumes:
+  postgres-data:
+
+services:
+  app:
+    container_name: javadev
+    build: 
+      context: .
+      dockerfile: Dockerfile
+      args:
+        # Update 'VARIANT' to pick an version of Java: 11, 17.
+        # Append -bullseye or -buster to pin to an OS version.
+        # Use -bullseye variants on local arm64/Apple Silicon.
```

nit: Do we need the "local" qualifier? 
```suggestion
        # Use -bullseye variants on arm64/Apple Silicon.
```

---

Well, that line of text was actually taken verbatim from the `docker-compose.yml` in the [python-3-postgres folder](https://github.com/microsoft/vscode-dev-containers/blob/main/containers/python-3-postgres/.devcontainer/)

---

I searched through the repository and the setup is listed as "local arm64/Apple Silicon" on all dockerfiles, so we can leave as-is.